# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libnvme/no-examples.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libnvme-1.14/meson.build.vanilla	2025-07-24 20:53:25.939896550 +0200
+++ libnvme-1.14/meson.build	2025-07-24 20:53:32.422487795 +0200
@@ -282,7 +282,7 @@
 if get_option('tests')
     subdir('test')
 endif
-subdir('examples')
+#subdir('examples')
 subdir('doc')
 
 ################################################################################
